src/metabase/pulse/render/js_svg.clj

```diff
@@ -55,27 +56,30 @@ const dimension_accessors = {
   metric: (row) => row[1],
 }
 
-function timeseries_line (data, labels) {
+function timeseries_line (data, labels, settings) {
```

Comment:
instead of embedding all this JavaScript in a giant string inline in the Clojure file, would it make sense to put it in a file like `src/metabase/pulse/render/js_svg.js`, and then just do `(slurp (io/resource "metabase/pulse/render/js_svg.js"))`?